pretrain/redpajama.py

```diff
@@ -157,20 +169,19 @@ def train(
         fabric.print(f"Measured TFLOPs: {measured_flops * fabric.world_size / 1e12:.2f}")
         del meta_model, x
 
-    step_count = 0
     total_lengths = 0
```

Comment:
Shouldn't we save this too?

Reply:
I wouldn't know why the speed monitor needs to be resumable. It can give a speed estimate regardless of the global iteration.